src/widgets/settingspages/GeneralPage.cpp

```diff
@@ -325,6 +325,8 @@ void GeneralPage::initLayout(GeneralPageView &layout)
                            "Google",
                        },
                        s.emojiSet);
+    layout.addCheckbox("Remove spaces around emotes",
+                       s.removeSpacesBetweenEmotes);
```

Comment:
How about moving this right below all other 4 checkboxes in the `Emotes` category? Also, thoughts on changing word "around" to "between" (this would also keep consistency with your setting from `Settings.hpp`)?

Reply:
ad83919